tests/basic/pool_function_test.go

```diff
@@ -606,3 +606,63 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
 	})
 
 })
+
+var _ = Describe("{PoolExpandResizePoolMaintenanceCycle}", func() {
+
+	var testrailID = 34542842
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542842
```

Comment:
This test case in test rail talks about cycle through maintenance mode after `add-disk` operation. Current test does it after `resize-disk`. Pls point to correct test in testrail.